README.md

```diff
@@ -19,27 +19,38 @@ These upgrade pipelines are intended to be kept running for as long as the found
 
 These pipelines are found in any of the directories with the `upgrade-` prefix.
 
+## Prerequisites 
+
+- [install a Concourse server](https://concourse.ci/installing.html)
+- download the [Fly CLI](https://concourse.ci/fly-cli.html) to interact with the Concourse server
+- depending on where you've installed Concourse, you may need to set up
+[additional firewall rules](FIREWALL.md "Firewall") to allow Concourse to reach
+third-party sources of pipeline dependencies
+
 ## Usage
 
-You'll need to [install a Concourse server](https://concourse.ci/installing.html)
-and get the [Fly CLI](https://concourse.ci/fly-cli.html)
-to interact with that server.
+1. We recommend downloading pcf-pipelines from [Pivotal Networks](https://network.pivotal.io/products/pcf-automation).
```

Comment:
This isn't just a "we recommend." If people don't get them from Pivnet they have to do extra work to use them without them breaking spontaneously. The only people that should clone the pipelines from Github are those that are extending them or making PRs against it.